packages/machine-extractor/src/MachineExtractResult.ts

```diff
@@ -1334,11 +1335,14 @@ export class MachineExtractResult {
 
           if (n.ArrayExpression.check(transitionProp.value)) {
             if (transitionProp.value.elements.length === 1) {
+              // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
But a lot of errors are disabled because that's currently the only way to make them stop without a lot of effort, like this unsafe assignment where we cast `minifiedTransition as any`.

To me, these cases are the main point of this PR. We're frequently using unsafe code in important parts of the codebase and that makes it difficult to maintain.

Manually resolving these errors is probably worth spending a good amount of time on, and now we can easily track them down by removing the comments associated with them.